kubernetes_platform/proto/kubernetes_executor_config.proto

```diff
@@ -0,0 +1,109 @@
+// Copyright 2023 The Kubeflow Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package kfp_kubernetes;
+
+import "google/protobuf/struct.proto";
+
+option go_package = "github.com/kubeflow/pipelines/kubernetes_platform/proto/go";
+
+message KubernetesExecutorConfig {
+    repeated SecretAsVolume secret_as_volume = 1;
+    repeated SecretAsEnv secret_as_env = 2;
+    repeated PvcMount pvc_mount = 3;
+}
+
+message SecretAsVolume {
+    // name of the Secret

```

Comment:
Nit: just to be consistent with the API protos, let's start with an upper case and end with a period mark. Also, I'm not sure if we need to start `Secret` with an upper case. Is there a specific reason you are doing so?

Reply:
Updated. Secret is capitalized since it refers to the specific [Kubernetes Secret object](https://kubernetes.io/docs/concepts/configuration/secret/).